hack/make-rules/test-cmd.sh

```diff
@@ -1569,6 +1569,9 @@ __EOF__
   kube::test::get_object_assert 'pods --namespace=other' "{{range.items}}{{$id_field}}:{{end}}" 'valid-pod:'
   # Post-condition: verify shorthand `-n other` has the same results as `--namespace=other`
   kube::test::get_object_assert 'pods -n other' "{{range.items}}{{$id_field}}:{{end}}" 'valid-pod:'
+  # Post-condition: a resource cannot be retrieved by name across all namespaces
+  output_message=$(! kubectl get pod valid-pod --all-namespaces 2>&1)
```

Comment:
always pass `"${kube_flags[@]}"` as well

see https://github.com/k82cn/kubernetes/blob/30cb370f5a7b9ce1e28499d89d77f2b10d98692c/hack/make-rules/test-cmd.sh#L797-L799 as an example for testing an expected failure and error message